VST/tests/vst_effect_test.ck

```diff
@@ -0,0 +1,24 @@
+"C:/VSTPlugIns/ValhallaRoom_x64.dll" => string vst_effect_path; // Must be absolute path.
```

Comment:
Are there MIT-licensed VST effect(s) we could use here instead of proprietary ones?

Reply:
What do you think of the 64-bit Dimension Expander plugin here? https://xferrecords.com/freeware/ It's labeled freeware and is working with my chuck test. Would it be ok to add the DLL to the repo?

Reply:
I've added the DLL in this commit https://github.com/ccrma/chugins/pull/47/commits/79c3e59901c3cfc99e0c35775bc478fcd343d5f3